<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>220</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>287</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>287</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>240</width>
     <height>186</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Server:</string>
        </property>
        <property name="buddy">
         <cstring>lineEditEtradeServerName</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>ID:</string>
        </property>
        <property name="buddy">
         <cstring>lineEditId</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>PassWord:</string>
        </property>
        <property name="buddy">
         <cstring>lineEditPassword</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Server Type:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditEtradeServerName">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditId"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditPassword">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBoxServerType">
        <property name="editable">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>real server</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>demo server</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonLogin">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>283</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonTest">
      <property name="text">
       <string>Test</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditMessage">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditEtradeServerName</tabstop>
  <tabstop>lineEditId</tabstop>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>comboBoxServerType</tabstop>
  <tabstop>pushButtonLogin</tabstop>
  <tabstop>lineEditMessage</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonLogin</sender>
   <signal>clicked()</signal>
   <receiver>dialog</receiver>
   <slot>slot_login()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonTest</sender>
   <signal>clicked()</signal>
   <receiver>dialog</receiver>
   <slot>slot_test()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_login()</slot>
  <slot>slot_test()</slot>
 </slots>
</ui>
